src/index.css

```diff
@@ -17,11 +18,11 @@
 
 .footer-text {
   text-transform: capitalize;
+  text-align: center;
 }
 
 main {
   min-height: calc(100vh - 4rem);
-
   display: flex;
```

Comment:
You already define a class as flex on line number 9. I kindly suggest that, use that class for your 'display: flex' properties to have dry and clean code. You could make this change in all classes in which you use 'display: flex'
It's like '.card-body flex' in line 49